package.json

```diff
@@ -13,35 +13,37 @@
   "dependencies": {
     "@babel/core": "^7.20.12",
     "@babel/eslint-parser": "^7.19.1",
-    "@next/eslint-plugin-next": "^13.1.1",
-    "@primer/css": "^20.4.0",
-    "@primer/react": "^35.4.0",
-    "@solana/spl-token": "^0.2.0",
-    "@solana/wallet-adapter-base": "^0.9.5",
-    "@solana/wallet-adapter-react": "^0.15.5",
-    "@solana/wallet-adapter-react-ui": "^0.9.7",
-    "@solana/wallet-adapter-wallets": "^0.16.1",
-    "@solana/web3.js": "^1.44.0",
-    "@typescript-eslint/eslint-plugin": "^5.48.0",
+    "@next/eslint-plugin-next": "^13.1.5",
+    "@primer/css": "^20.8.0",
+    "@primer/octicons-react": "^17.10.2",
+    "@primer/react": "^35.17.0",
+    "@solana/spl-token": "^0.3.7",
+    "@solana/wallet-adapter-base": "^0.9.20",
+    "@solana/wallet-adapter-react": "^0.15.28",
+    "@solana/wallet-adapter-react-ui": "^0.9.27",
+    "@solana/wallet-adapter-wallets": "^0.19.10",
+    "@solana/web3.js": "^1.73.0",
+    "@typescript-eslint/eslint-plugin": "^5.49.0",
     "deepmerge": "^4.2.2",
     "husky": "^8.0.3",
     "lint-staged": "^13.1.0",
-    "mongoose": "^6.4.7",
-    "next": "12.1.6",
-    "react": "18.1.0",
-    "react-dom": "18.1.0",
-    "sass": "^1.54.0",
+    "mongoose": "^6.8.4",
+    "next": "13.1.5",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "sass": "^1.57.1",
     "styled-components": "^5.3.5"
```

Comment:
could this be updated? 
![image](https://user-images.githubusercontent.com/72078237/214315418-8ac7bd1e-59eb-442e-96e2-79af02a8b66e.png)